.gitignore

```diff
@@ -23,4 +23,16 @@ go.work.sum
 
 # env file
 .env
-.idea
\ No newline at end of file
+.idea
+
+# Node.js dependencies
+node_modules/
+
+# Playwright test results and reports
+test-results/
+playwright-report/
+screenshots/
```

Comment:
For some reason we have debug png screenshots that have been committed. Remove the debug screenshots